pallets/external-validator-slashes/src/mock.rs

```diff
@@ -196,6 +200,57 @@ impl DeferPeriodGetter {
     }
 }
 
+pub struct MockOkOutboundQueue;
+impl SendMessage for MockOkOutboundQueue {
+    type Ticket = ();
+
+    fn validate(
+        _: &snowbridge_core::outbound::Message,
+    ) -> Result<(Self::Ticket, Fee<Self::Balance>), SendError> {
+        Ok((
+            (),
+            Fee {
+                local: 1,
+                remote: 1,
+            },
+        ))
+    }
+
+    fn deliver(_: Self::Ticket) -> Result<H256, SendError> {
+        Ok(H256::zero())
+    }
+}
+
+impl SendMessageFeeProvider for MockOkOutboundQueue {
+    type Balance = u128;
+
+    fn local_fee() -> Self::Balance {
+        1
+    }
+}
+pub struct MockErrOutboundQueue;
+impl SendMessage for MockErrOutboundQueue {
+    type Ticket = ();
+
+    fn validate(
+        _: &snowbridge_core::outbound::Message,
+    ) -> Result<(Self::Ticket, Fee<Self::Balance>), SendError> {
+        Err(SendError::MessageTooLarge)
+    }
+
+    fn deliver(_: Self::Ticket) -> Result<H256, SendError> {
+        Err(SendError::MessageTooLarge)
+    }
+}
+
+impl SendMessageFeeProvider for MockErrOutboundQueue {
```

Comment:
impled twice, one for MockErrOutboundQueue and the other for MockOkOutboundQueue. MockErrOutboundQueue is not used anymore right?

Reply:
No, I copied this from snowbridge, will remove it.